src/candela/components/LineUp/index.js

```diff
@@ -83,35 +101,41 @@ export default class LineUp {
   * @returns: relative scale of lineup to available space.
   */
   createLineupAdjustWidth (elem, name, spec, fixed) {
-    let total = 0;
-    let count = 0;
-    let c1;
-    let c2;
-    // let width = $(elem)[0].getBoundingClientRect().width - $.scrollbarWidth() - fixed;
-    let width = $(elem)[0].getBoundingClientRect().width - 30 - fixed;
-    let col = spec.dataspec.layout.primary;
-
+    var rankWidth = 0;
```

Comment:
Minor style thing but I'd prefer the use of `let` everywhere rather than `var` unless there's a semantics reason to prefer `var` here.